identity-tests/src/main/scala/com/gu/identity/integration/test/pages/FaceBookAuthDialog.scala

```diff
@@ -0,0 +1,43 @@
+package com.gu.identity.integration.test.pages
+
+import com.gu.integration.test.pages.common.ParentPage
+import org.openqa.selenium.support.ui.{WebDriverWait, ExpectedConditions}
+import org.openqa.selenium.{By, WebDriver, WebElement}
+
+class FaceBookAuthDialog(implicit driver: WebDriver) extends ParentPage {
```

Comment:
If you need to use WebDriver you may be able to do it more nicely using the [ScalaTest WebBrowser DSL](http://www.scalatest.org/user_guide/using_selenium)